python/hail/docs/guides/genetics/genetics.rst

```diff
@@ -0,0 +1,127 @@
+Genetics
+========
+
+Formatting
+~~~~~~~~~~
+
+..
```

Comment:
what is this?

Reply:
ah, this was sort of hacky. it sets up the table properly for the next doctest below this.

I needed a table with a field `variant` so that the doctest for `ht.key_by(**hl.parse_variant(ht.variant))` (which is below this) would pass. All the doctests currently use conftest.py as their set up, and I couldn't figure out how to add a python fixture for a single *.rst file, so I just added the variant field here instead. I can reinvestigate individual fixtures for *.rst files if that would be better.